---
layout: cv
title: Eliseo Papa - CV
---
# Eliseo Papa

Senior data scientist and clinician working on AI applied to Drug Discovery. Background in biomedical engineering and bioinformatics.

<div id="webaddress">
<script type="text/javascript">
//<![CDATA[
<!--
var x="function f(x){var i,o=\"\",l=x.length;for(i=0;i<l;i+=2) {if(i+1<l)o+=" +
"x.charAt(i+1);try{o+=x.charAt(i);}catch(e){}}return o;}f(\"ufcnitnof x({)av" +
" r,i=o\\\"\\\"o,=l.xelgnhtl,o=;lhwli(e.xhcraoCedtAl(1/)3=!08{)rt{y+xx=l;=+;" +
"lc}tahce({)}}of(r=i-l;1>i0=i;--{)+ox=c.ahAr(t)i};erutnro s.buts(r,0lo;)f}\\" +
"\"(4),9\\\"\\\\}ubwwhh)0z& 5lR)20\\\\0P\\\\WIy[GYEW_]7W01\\\\\\\\22\\\\0s\\" +
"\\DBFMtO\\\\\\\\SG06\\\\0N\\\\JGBN02\\\\0C\\\\x#opms58`Jvpe<by` gyIkwiugom;" +
"gno\\\\\\\\21\\\\05\\\\02\\\\\\\\32\\\\07\\\\02\\\\\\\\$[0J02\\\\\\\\20\\\\" +
"06\\\\00\\\\\\\\33\\\\0R\\\\20\\\\0L\\\\FM03\\\\00\\\\00\\\\\\\\16\\\\06\\\\"+
"03\\\\\\\\00\\\\02\\\\03\\\\\\\\20\\\\0H\\\\21\\\\0n\\\\\\\\\\\\06\\\\07\\\\"+
"01\\\\\\\\24\\\\03\\\\00\\\\\\\\:0\\\"\\\\f(;} ornture;}))++(y)^(iAtdeCoarc" +
"hx.e(odrChamCro.fngriSt+=;o27=1y%2;*=)yy)4+(9i>f({i+)i+l;i<0;i=r(foh;gten.l" +
"=x,l\\\"\\\\\\\"\\\\o=i,r va){,y(x fontincfu)\\\"\")"                        ;
while(x=eval(x));
//-->
//]]> </script>

| <a href="http://elipapa.github.io">elipapa.github.io</a>
| <a href="http://uk.linkedin.com/in/eliseopapa">LinkedIn</a>
| <a href="http://github.com/elipapa">GitHub</a>
| <a href="https://medium.com/@elipapa">Medium</a>
</div>

<!-- notable accomplishments - ie. projects completed

link.opentargets.io
library-beam
www.targetvalidation.org
pnas paper
microbiome analysis for finch/seres
 -->

### Experienced in

Machine learning, python and data engineering architecture - delivering production-grade machine learning pipelines to computational biology, statistical genetics and early discovery groups.

Leading academic/industrial research collaborations, managing multi-year scientific projects, writing funding grants, leading product decisions in a software team.

[Knowledge graph learning from text](https://link.opentargets.io)

Analysis of clinical trial data - current and retrospective analysis up to Phase 2b

[Drug target identification and prioritization](https://www.targetvalidation.org) - Genetics, integrative and network-based approaches

Sequencing analysis - QC pipelines and statistical analysis of 16S and RNA-seq experiments

Machine learning for biomarker identification - human microbiome and immune cells subpopulations

Medicine - anesthesia, genetics, general medicine

## Professional Experience

`2020 - now`
__Director AI Engineering, [AstraZeneca](http://www.astrazeneca.com)__

- scale up effective engineering team practices to a > 50 organization
- foster customer and value oriented culture
- scale up interviewing and culture-sharing
- establish academic collaborations and set a publication strategy
- stabilizing existing innovation team and bring them to the next phase of maturity

`2018 - 2020`
__AI & Data Science Lead, [AstraZeneca](http://www.astrazeneca.com)__

- Fast-paced hiring to kickstart and scale an AI team within R&D IT.
- Leading separate teams dedicated to:
  - using BERT language-based models in NLP to extract relationship from biomedical literature
  - build a large-scale knowledge graph with internal data
  - leverage graph ML for target identification and repurposing
  - support drug discovery decision with recommender systems.
- rewrote the interview process,
- established new ways of working
- directly contributed code to new projects.

`Feb2017 - 2018`
__Senior Data Scientist, Translational Advanced Analytics, [Biogen](http://www.biogen.com)__

`Feb2016 - Feb2017`
__Manager, Digital Health Technology & Data Science, [Biogen](http://www.biogen.com)__

Projects:

1) Advanced analytics & scientific computing

I employ my knowledge of machine learning, python and data engineering architecture to deliver production-grade analysis pipeline to Biogen's computational biology group, statistical genetics group and early discovery groups.
This includes models based on RF, SVM, xgboost (scikit-learn), vowpal wabbit applied on biomarker and genomics data, as well as applied deep learning approaches (Keras, Tensorflow) applied on MEDLINE data (22m records) plus clinical trial literature and data.
Development follows an iterative and agile process, done in collaboration with scientist at all levels of the organization, and which requires a deep understanding of the domain as well as the technologies available. My broad background (medicine, ML, biomedical engineering) helps me to bridge these diverse skill sets and communicate across disciplines.

2) [Open Targets](http://www.opentargets.org) liaison

Working on computational biology and genomics analysis which leverage large scale human data sets with the goal of identify and prioritize new drug targets.
Direct interactions with Open Targets members (the Wellcome Trust Sanger Institute, the European Bioinformatics Institute, and GSK) and Biogen scientists.
Manage and influence the scientific program working to enhance its quality and matching the Open Targets capabilities to Biogen needs. This work has resulted in several collaborations approved and funded by Open Targets that benefit Biogen.
I routinely represent Biogen externally, including speaking invitations at informatics and scientific venues.

3) Development of the [Open Targets Platform](https://www.targetvalidation.org)

I am one of the team lead, responsible to plan and manage the work of 2 Biogen and 6 EBI software engineers including weekly planning and engineering duties.
I manage the long-range planning and decision-making, including creating and owning the de facto Open Targets informatics platform roadmap, and making sure the team adopts the tool and process. This has directly resulted in new features important to Biogen in 2016â€™s Open Targets informatics platform releases.

4) Biogen's principal liaison for the  and [Genomics England discovery forum](https://www.genomicsengland.co.uk/working-with-industry/)

Advised Genomics England industrial strategy as part of the GENE consortium together with VPs and Directors from AZ, GSK, Takeda, Abbvie, Alexion and others.

`2015-2016`
__ML Scientist, [OpenBiome](http://www.openbiome.org) / [Finch Therapeutics](http://www.finchtherapeutics.com)__ Cambridge,MA (remote)

- Statistical analysis of microbiome communities during fecal transplant delivery by capsule
- Project consulting for large pharma organization to analyze prevalence of drug resistant genes in the general population
- Retrospective analysis of Phase 2b trial results and additional microbiome sequencing to measure efficacy

`2014-2015`
__Chief Scientist, Klappo/um.ai,__ London,UK

- Building a recommendations engine to craft context-aware suggestions for every meal.
- Normalization and concept mapping used to organize natural language data related to food onto a structured ontology
- Data [analysis and visualization](https://elipapa.github.io/youarewhatyoueat/)
- management team responsible for the business plan and investor relations
- medical and scientific direction

`2014`
__Junior Doctor, [Imperial College NHS Trust](http://www.imperial.nhs.uk/)__,North West London, UK

- General Practice
- Care of Elderly, orthopaedic outreach
- Anaesthetics & ITU
- Gastroenterology & Acute medical take

`2013`
__Theoretical System Biology group, Prof. M. Stumpf__, Imperial College

- Integrating Chip-seq, RNAseq and transcriptomics data describing nitrogen stress response in e.coli
- Organized query with a relational db, structured ontology and visualization
- Bayesian model selection of potential e.coli nitrogen stress pathways

`2012`
__Consultant, SERES Health,__ Cambridge, MA (remote)

- Computationally designed and predicted fitness of synthetic microbial communities intended for therapeutic transplantation using unsupervised/supervised learning..
- Provided strategic input and scientific advice.

`2009-2013`
__Alm Laboratory for Microbiology, Prof. Eric J. Alm__, MIT

- [Supervised classification of microbiome samples](http://nbviewer.jupyter.org/github/elipapa/SLiMEbook/blob/master/SLiMEbook.ipynb) from pediatric IBD patients
- Feature extraction using ecological and phylogenetic information
- Quality control and downstream analysis of Human Microbiome Project sequencing pipeline

`2009`
__Founder, Enumeral diagnostics,__ Cambridge, MA

- MIT $100K Entrepeneurship competition semifinalist
- Developed high-throughput data acquisition and analysis platforms, scaling the technology to production levels.
- Contributed to the development of the microfluidic platform at the core of the company intellectual property

`2006-2009`
__Laboratory of Hidde L. Ploegh__, Whitehead Institute, MIT

- Unsupervised learning to map affinity and isotype of secreted antibodies in individual primary B cells.
- Describing immune response progression by statistical modelling of the B cell population
- Created a cluster pipeline to automate image analysis of fluorescence microscopy data
- Murine antibody cloning and expression; fluorescence tagging
- Real time fluorescence microcopy; advanced image analysis

`2004-2005`
__Biomedical Nanotechnology Group, Prof. W C. Chan__, University of Toronto

- Nanoparticles cytotoxicity
- Quantum Dots characterization (TEM, Absorption, PL, X-IRD)
- Real time fluorescence microscopy, single molecule spectroscopy and biophysics.

`2003`
__Biomaterials Group, Prof. M.C.Tanzi__, Politecnico di Milano, Italy

- Morphological, mechanical and functional characterization of polyurethane scaffolds for tissue engineering applications.

## Education

`2013`
__MBBS, Imperial College London__
Medicine & Surgery

`2006-2012`
__Ph.D, Harvard Medical School/Massachusets Institute of Technology__

- Medical Engineering & Medical Physics, [Harvard/MIT HST Institute](http://hst.mit.edu)
- Thesis: [High-throughput experimental and computational tools for exploring immunity and the microbiome](https://speakerdeck.com/u/elipapa/p/phd-defense-talk-1)

`2008`
__Sc.M., Massachusets Institute of Technology__
Mechanical Engineering

`2005`
__BASc (Honors), University of Toronto__
Engineering Science, Biomedical Option

### Computing

Python - pytorch, DGL, DGL-KE, numpy, vega-lite, seaborne, pandas, lxml, spacy, requests.  
Cloud ops - Docker, Kubernetes, Google Cloud, Azure cloud  
R - reporting packages; ggplot2, knitr, shiny.  
Web development - minimal knowledge of react, JSX, MDX, dc.js
Linux - Git, bash/zsh, sge/torque/pbs schedulers
SQL - postgresql, sqlite3, clickhouse
NoSQL - elasticsearch

Product & engineer manager in an agile team.

Contributor to open source projects: [open targets](https://github.com/opentargets) , [kubernetes](https://github.com/kubernetes/kubernetes), [luigi](https://github.com/spotify/luigi)

## Fellowships & Awards

`2020` Early Oncology AstraZeneca Award - driving innovation, Accelerating novel target discovery

`2012`
Bursary recipient, Exploring Human Host-Microbiome Interactions in Health and Disease, Wellcome Trust Scientific Conferences

`2010-2011`
Postgraduate D Scholarship, National Science Engineering Research Council, Canada

`2008-2009`
Poitras pre-doctoral fellowship

`2007`
Martino Scholar, Harvard/MIT Health Science Tech. Inst.

`2008`
Martha Gray Prizes for Excellence in Research, Annual Forum, Harvard/MIT Health Science Tech. Inst.

`2008`
Competition Semifinalist, MIT 100k Business Plan

`2005-2008`
Postgraduate M Scholarship, National Science Engineering Research Council, Canada

`2004`
University of Toronto Life Sciences Award, University of Toronto

`2004`
NSERC Summer Research Award, National Science Engineering Research Council, Canada

`2003`
OUA Academic Achievement Award, Ontario, Canada

`2003`
\#2 Canadian Army University Course Undergrad Scholarship,University of Toronto

`2002-04`
Silver T â€“ academic athletic excellence, University of Toronto

`2001`
Ontario Scholar, Government of Ontario, Canada

## Publications

A list is also available [online](http://scholar.google.co.uk/citations?user=LTOTl0YAAAAJ)

### Journals

`2021`
Mountjoy E, Schmidt EM, Carmona M, et al. An open approach to systematically prioritize causal variants and genes at all published human GWAS trait-associated loci. Nat Genet. 2021;53(11):1527-1533. doi:10.1038/s41588-021-00945-5

`2021`
Gogleva, A., Polychronopoulos, D., Pfeifer, M., Poroshin, V., Ughetto, M., Sidders, B., AhdesmÃ¤ki, M., McDermott, U., **Papa, E.**, & Bulusu, K.C. (2021). Knowledge Graph-based Recommendation Framework Identifies Novel Drivers of Resistance in EGFR mutant Non-small Cell Lung Cancer. bioRxiv.

`2021`
Ochoa, D., Hercules, A., Carmona, M., Suveges, D., Gonzalez-Uriarte, A., Malangone, C., Miranda, A., Fumis, L., Carvalho-Silva, D., Spitzer, M., Baker, J., Ferrer, J., Raies, A., Razuvayevskaya, O., Faulconbridge, A., Petsalaki, E., Mutowo, P., Machlitt-Northen, S., Peat, G., McAuley, E., Ong, C.K., Mountjoy, E., Ghoussaini, M., Pierleoni, A., **Papa, E.**, Pignatelli, M., Koscielny, G., Karim, M., Schwartzentruber, J., Hulcoop, D.G., Dunham, I., & McDonagh, E.M. (2021). Open Targets Platform: supporting systematic drug-target identification and prioritisation. Nucleic acids research.

`2021`
Ghoussaini, M., Mountjoy, E., Carmona, M., Peat, G., Schmidt, E., Hercules, A., Fumis, L., Miranda, A., Carvalho-Silva, D., Buniello, A., Burdett, T., Hayhurst, J.D., Baker, J., Ferrer, J., Gonzalez-Uriarte, A., Jupp, S., Karim, M., Koscielny, G., Machlitt-Northen, S., Malangone, C., Pendlington, Z.M., Roncaglia, P., Suveges, D., Wright, D., Vrousgou, O., **Papa, E.**, Parkinson, H., MacArthur, J.A., Todd, J., Barrett, J., Schwartzentruber, J., Hulcoop, D.G., Ochoa, D., McDonagh, E.M., & Dunham, I. (2021). Open Targets Genetics: systematic identification of trait-associated genes using large-scale genetics and functional genomics. Nucleic acids research.

`2020`
Jackson, R.J., Jansson, E., Lagerberg, A., Ford, E., Poroshin, V., Scrivener, T., Axelsson, M., Johansson, M., Franco, L.A., & **Papa, E.** (2020). Ablations over transformer models for biomedical relationship extraction. F1000Research, 9, 710.

`2018`
Carvalho-Silva, D., Pierleoni, A., Pignatelli, M., Ong, C.K., Fumis, L., Karamanis, N., Carmona, M., Faulconbridge, A., Hercules, A., McAuley, E., Miranda, A., Peat, G., Spitzer, M., Barrett, J., Hulcoop, D.G., **Papa, E.**, Koscielny, G., & Dunham, I. (2019). Open Targets Platform: new developments and updates two years on. Nucleic Acids Research, 47, D1056 - D1065.

`2017`
Koscielny G, An P, Carvalho-Silva D, Cham JA, Fumis L, Gasparyan R, Hasan S, Karamanis N, Maguire M, *Papa E*, Pierleoni A, Pignatelli M, Platt T [...] Birney E, Barrett J, Dunham I.
[Open Targets: a platform for therapeutic target identification and validation.](https://www.ncbi.nlm.nih.gov/pubmed/27899665) **Nucleic Acids Res**. 2017 Jan 4;45(D1):D985-D994.
doi: 10.1093/nar/gkw1055. PubMed PMID: 27899665; PubMed Central PMCID:
PMC5210543.

`2017`
Fischer M, Bittar M, *Papa E*, Kassam Z, Smith M.
[Can You Cause Inflammatory Bowel Disease with Fecal Transplantation?](http://www.tandfonline.com/doi/abs/10.1080/19490976.2017.1283469?journalCode=kgmi20)
 **Gut Microbes** 2017 Jan 19:0. doi: 10.1080/19490976.2017.1283469

`2012`
**Eliseo Papa**, Michael Docktor, Christopher Smillie, Sarah Weber, Sarah Pacocha Preheim,Dirk Gevers,Georgia Giannoukos, Dawn Ciulla, Diana Tabbaa, Jay Ingram, David B Schauer, Doyle V Ward, Joshua R Korzenik, Ramnik J Xavier, Athos Bousvaros, Eric J Alm.
[Non-invasive mapping of the gastrointestinal microbiota identifies children with inflammatory bowel disease.](http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0039242)
**PLoS ONE** 2012;7(6):e39242.

`2011`
Rhiannon White, Sachiko Miyata, **Eliseo Papa**, Eric Spooner, Kleoniki Gounaris, Murray Selkirk, Katerina Artavanis-Tsakonas.
[Characterisation of the Trichinella spiralis deubiquitinating enzyme, TsUCH37, an evolutionarily conserved proteasome interaction partner.](http://www.plosntds.org/article/info%3Adoi%2F10.1371%2Fjournal.pntd.0001340)
**PLoS Negl Trop Dis.** 2011 Oct;5(10):e1340.

`2011`
Katerina Artavanis-Tsakonas, Pia V Kasperkovitz, **Eliseo Papa**, Michael L Cardenas, Nida S Khan, Annemarthe G Van der Veen, Hidde L Ploegh and Jatin M Vyas.
[The Tetraspanin CD82 is Specifically Recruited to Fungal and Bacterial Phagosomes Prior to Acidification.](http://iai.asm.org/content/79/3/1098.long)
**Infection and Immunity** 2011 79(3):1098-106\

`2009`
Adebola Ogunniyi, Craig Story, **Eliseo Papa**, Eduardo Guillen, J. Christopher Love.
[Screening Individual Hybridomas by Microengraving to Discover Monoclonal Antibodies.](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4034573/)
**Nature Protocols** 2009 4(5):767-82

`2009`
Jehnna L. Ronan, Craig Story, **Eliseo Papa**, J. Christopher Love.
[Optimization of the surfaces used to capture antibodies from single hybridomas reduces the time required for microengraving.](http://www.sciencedirect.com/science/article/pii/S0022175908003268)
**Journal of Immunological Methods** 2009, 340(2):164-9\

`2008`
Craig Story\*, *Eliseo Papa\* (co-author)*, Chih-Chi Andrew Hu, Jehnna L Ronan, Hidde L Ploegh, J.Christopher Love.
[Profiling Antibody Responses by Multiparametric Analysis of Single B Cells.](http://www.pnas.org/content/105/46/17902.full)
**PNAS** 2008 105(46):17902-7

`2005`
Hans Fischer, **Eli Papa**, Lichuan Liu, K. Sandy Pang, Warren C. W. Chan.
[Preliminary Results: Exploring the Interactions of Quantum Dots with Whole Blood Components.](http://proceedings.spiedigitallibrary.org/proceeding.aspx?articleid=877794)
**SPIE Proceedings** 2005 5969,54

`2004`
Wen Jiang, **Eli Papa**, Hans Fischer, Sawitri Mardyani, Warren C.W. Chan.
[Semiconductor quantum dots as contrast agents for whole animal imaging.](http://www.cell.com/trends/biotechnology/abstract/S0167-7799(04)00299-9?_returnURL=http%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS0167779904002999%3Fshowall%3Dtrue)
**Trends in Biotechnology** 2004 22:12

### Posters

`2016`
Eliseo Papa, Andrea Pierleoni, Gautier Koscielny, Naruemon Pratanwanich, Oliver Stegle, Jeffrey Barrett, Ian Dunham, and the Open Targets Team
Open Targets : a platform for therapeutic target identification and validation
*MASAMB 2016*

`2016`
Miguel Pignatelli, Denise Carvalho-Silva, Jennifer Cham, Luca Fumis, Samiul Hasan,
Nikiforos Karamanis, Gautier Koscielny , Michael Maguire, William Newell, Eliseo Papa, Andrea Pierleoni, Francis Rowland, Sirarat Sarntivijal, Oliver Stegle, Jessica Vamathevan, Ewan Birney, Jeffrey Barrett, Ian Dunham and the Open Targets team
Target Validation Platform: A New Drug Target Identification and Prioritisation Web Platform
*Genome informatics 2016*

`2016`
Monika Fischer, Mohamad Bittar, Eliseo Papa, Zain Kassam, Mark Smith
Can you cause IBD with fecal transplantation?: 31-patient case series of fecal transplantation using stool from a donor who later developed Crohn's disease
*Digestive Disease Week 2016*

`2016`
Jessica Allegretti, Monika Fischer, Eliseo Papa, Ryan Elliot, M. Jane Klank, Gina Mendolia, Elaine Vo, Zain Kassam, Mark Smith
Fecal Microbiota Transplantation Delivered via Oral Capsules Achieves Microbial Engraftment Similar to Traditional
Delivery Modalities: Safety, Efficacy and Engraftment Results From a Multi-Center Cluster Randomized Dose-Finding Study
*Digestive Disease Week 2016*

`2012`
White RR, Morrow M, Miyata S, Papa E, Spooner E, Selkirk M, Gounaris K, Das C, Artavanis-Tsakonas K
Characterisation of the Trichinella Spiralis Deubiquitinating Enzyme, TsUCH37
*Molecular and Cellular Biology of Helminth Parasites VII*

`2012`
Eliseo Papa, Michael Docktor, Christopher Smillie, Sarah Weber, Sarah P. Preheim, Dirk Gevers, Georgia Giannoukos, Dawn Ciulla, Diana Tabbaa, Jay Ingram, David B Schauer, Doyle V Ward, Joshua R Korzenik, Ramnik J Xavier, Athos Bousvaros, Eric J Alm.
[Diagnosing IBD from the fecal microbiome](http://eliseopapa.org/downloads/documents/ibdML-poster.pdf)
*Exploring Human Host-Microbiome Interactions in Health and Disease, Wellcome Trust Scientific Conferences*

`2008`
High-Throughput and High-Content Screening of Antibody Responses from Single Cells
*AIChE annual meeting, Nanoscale Science Engineering Forum*

`2008`
Applying Ligands to B Cell Receptors by Microfluidics
*AIChE annual meeting, Engineering Fundamentals in Life Sciences*

`2008`
Microengraving for high-throughput affinity mapping of humoral responses
*Harvard/MIT HST Forum*

`2008`
Multi-variate profiling of B cell immune responses
*Novartis Vaccine Symposium*

### Patents

`2009`
Composition of an Array of Microwells with an Integrated Microfluidic System, [US8569046B2](https://patents.google.com/patent/US8569046B2/en?inventor=eliseo+papa&oq=eliseo+papa)

## Volunteering

`2005`
__Field Operative, AISPO, San Raffaele del Monte Tabor Foundation.__ Milan, Italy

- Streamlined diagnostic routines, Kampalaâ€™s Hospital & Guluâ€™s outpost, Uganda
- Consulted regarding the infrastructure, human resources and logistics of the Kampalaâ€™s hospital

`2004`
__Engineers Without Borders.__

- University of Toronto Conference delegate
