---
layout: cv
title: Eliseo Papa - Resume
---
# Eliseo Papa

Senior data scientist and clinician working on AI applied to Drug Discovery. Background in biomedical engineering and bioinformatics.

<div id="webaddress">
<script type="text/javascript">
//<![CDATA[
<!--
var x="function f(x){var i,o=\"\",l=x.length;for(i=0;i<l;i+=2) {if(i+1<l)o+=" +
"x.charAt(i+1);try{o+=x.charAt(i);}catch(e){}}return o;}f(\"ufcnitnof x({)av" +
" r,i=o\\\"\\\"o,=l.xelgnhtl,o=;lhwli(e.xhcraoCedtAl(1/)3=!08{)rt{y+xx=l;=+;" +
"lc}tahce({)}}of(r=i-l;1>i0=i;--{)+ox=c.ahAr(t)i};erutnro s.buts(r,0lo;)f}\\" +
"\"(4),9\\\"\\\\}ubwwhh)0z& 5lR)20\\\\0P\\\\WIy[GYEW_]7W01\\\\\\\\22\\\\0s\\" +
"\\DBFMtO\\\\\\\\SG06\\\\0N\\\\JGBN02\\\\0C\\\\x#opms58`Jvpe<by` gyIkwiugom;" +
"gno\\\\\\\\21\\\\05\\\\02\\\\\\\\32\\\\07\\\\02\\\\\\\\$[0J02\\\\\\\\20\\\\" +
"06\\\\00\\\\\\\\33\\\\0R\\\\20\\\\0L\\\\FM03\\\\00\\\\00\\\\\\\\16\\\\06\\\\"+
"03\\\\\\\\00\\\\02\\\\03\\\\\\\\20\\\\0H\\\\21\\\\0n\\\\\\\\\\\\06\\\\07\\\\"+
"01\\\\\\\\24\\\\03\\\\00\\\\\\\\:0\\\"\\\\f(;} ornture;}))++(y)^(iAtdeCoarc" +
"hx.e(odrChamCro.fngriSt+=;o27=1y%2;*=)yy)4+(9i>f({i+)i+l;i<0;i=r(foh;gten.l" +
"=x,l\\\"\\\\\\\"\\\\o=i,r va){,y(x fontincfu)\\\"\")"                        ;
while(x=eval(x));
//-->
//]]> </script>

| <a href="http://elipapa.github.io">elipapa.github.io</a>
| <a href="http://uk.linkedin.com/in/eliseopapa">LinkedIn</a>
| <a href="http://github.com/elipapa">GitHub</a>
| <a href="https://medium.com/@elipapa">Medium</a>
</div>

<!-- notable accomplishments - ie. projects completed

link.opentargets.io
library-beam
www.targetvalidation.org
pnas paper
microbiome analysis for finch/seres
 -->

### Experienced in

Machine learning, python and data engineering architecture - delivering production-grade analysis pipeline to computational biology, statistical genetics and early discovery groups.

Leading academic/industrial research collaborations, communicating across disciplines, managing multi-year scientific projects, writing funding grants, leading product decisions in a software team.

[Knowledge graph learning from text](https://link.opentargets.io)

Analysis of clinical trial data - current and retrospective analysis up to Phase 2b

[Drug target identification and prioritization](https://www.targetvalidation.org) - Genetics, integrative and network-based approaches

Sequencing analysis - QC pipelines and statistical analysis of 16S and RNA-seq experiments

Machine learning for biomarker identification - human microbiome and immune cells subpopulations

Medicine - anesthesia, genetics, general medicine

### Computing

Python - ipython, scikit-learn, numpy, matplotlib, seaborne, pandas, lxml, NLTK, requests, sqlite3.  
Cloud ops - Docker, Kubernetes, Google Cloud, AWS  
R - applied ML packages; ggplot2, knitr, shiny.  
Web development - HTML/CSS/JS, D3, dc.js, crossfilter.  
Linux - Git, bash/zsh, sge/torque/pbs  
SQL

Product & engineer manager in an agile team.

Contributor to open source projects: [open targets](https://github.com/opentargets), [kubernetes](https://github.com/kubernetes/kubernetes), [luigi](https://github.com/spotify/luigi), [airflow](https://github.com/apache/incubator-airflow/pulls)

## Work

`Oct2020 - now`
__Director AI Engineering, [AstraZeneca](http://www.astrazeneca.com)__

`Oct2018 - Oct2020`
__AI & Data Science Lead, [AstraZeneca](http://www.astrazeneca.com)__

- Fast-paced hiring to kickstart and scale an AI team within R&D IT.
- Leading separate teams dedicated to:
  - using BERT language-based models in NLP to extract relationship from biomedical literature
  - build a large-scale knowledge graph with internal data
  - pilot the use of graph learning representation and graph ML for target identification, repurposing and recommender systems.

I was hired to build a Data Science team focussed on building and exploiting knowledge graphs for target recommendations. Our aim is to make the job of the drug discovery scientist faster and more successful using AI/Machine learning and robust data science tools.
Joining a large corporation, it was imperative to build the right "maker" culture where the team could be given the autonomy to start small, build things fast and scale them. In less than six months, I rewrote the interview process, established new ways of working and directly contributed code to new projects. The team is now more than 15 people, regularly releasing the results of machine learning pipeline for the use of our scientists and participating in pipeline decisions. I learned a lot: from how to train NLP transformer models to interviewing effectively, managing data science projects in an applied research setting and run projects at scale.


`Feb2017 - Oct2018`
__Senior Data Scientist, Translational Advanced Analytics, [Biogen](http://www.biogen.com)__

`Feb2016 - Feb2017`
__Manager, Digital Health Technology & Data Science, [Biogen](http://www.biogen.com)__

Concurrent projects:

1) Advanced analytics & scientific computing

I employ my knowledge of machine learning, python and data engineering architecture to deliver production-grade analysis pipeline to Biogen's computational biology group, statistical genetics group and early discovery groups.
This includes models based on RF, SVM, xgboost (scikit-learn), vowpal wabbit applied on biomarker and genomics data, as well as applied deep learning approaches (Keras, Tensorflow) applied on MEDLINE data (22m records) plus clinical trial literature and data.

2) [Open Targets](http://www.opentargets.org) liaison

Working on computational biology and genomics analysis which leverage large scale human data sets with the goal of identify and prioritize new drug targets.
Manage and influence Open Targets scientific program, working in partnership with the Wellcome Trust Sanger Institute, the European Bioinformatics Institute, and GSK. This work has resulted in several collaborations approved and funded by Open Targets that benefit Biogen.
I routinely represent Biogen externally, including speaking invitations at informatics and scientific venues.

3) Development of the [Open Targets Platform](https://www.targetvalidation.org)

I am one of the team lead, responsible to plan and manage the work of 2 Biogen and 4 EBI engineers including weekly planning and engineering duties.
I manage the long-range planning and decision-making, including creating and owning the de facto Open Targets informatics platform roadmap, and making sure the team adopts the tool and process. This has directly resulted in new features important to Biogen in 2016â€™s Open Targets informatics platform releases.

4) Biogen's principal liaison for the  and [Genomics England discovery forum](https://www.genomicsengland.co.uk/working-with-industry/)

Participated and contributed to the Genomics England GENE consortium together with VPs and Directors from AZ, GSK, Takeda, Abbvie, Alexion and others.

`Apr2015 - Feb2016`
__ML Analyst, [OpenBiome](http://www.openbiome.org) / [Finch Therapeutics](http://www.finchtherapeutics.com)__ Cambridge,MA (remote)

- Statistical analysis of microbiome communities during fecal transplant delivery by capsule
- Project consulting for large pharmaceutical to analyze prevalence of drug resistant genes in the general population
- Retrospective analysis of Phase 2b trial results and additional microbiome sequencing to measure efficacy

`Sep2014-Jun2016`
__Chief Scientist, Klappo/um.ai,__ London,UK

- Building a recommendations engine to craft context-aware suggestions for every meal.
- Normalization and concept mapping used to organize natural language data related to food onto a structured ontology
- Data [analysis and visualization](https://elipapa.github.io/youarewhatyoueat/)
- Management team responsible for the business plan and investor relations
- Medical and scientific direction

`Aug2013-Dec2014`
__Junior doctor, [Imperial College NHS Trust](http://www.imperial.nhs.uk/)__, London, UK

- Anesthesia
- Care of the elderly, palliative care
- Gastroenterology

`Jun2013`
__Theoretical System Biology group, Prof. M. Stumpf__, Imperial College

- Integrating Chip-seq, RNAseq and transcriptomics data describing nitrogen stress response in e.coli
- Organized query with a relational db, structured ontology and visualization
- Bayesian model selection of potential e.coli nitrogen stress pathways

`May 2012 - Dec 2012`
__Consultant, [SERES Health](http://www.flagshipventures.com/venture-capital/all-companies/seres-health),__ Cambridge, MA

- Selection and fitness prediction of synthetic microbial communities intended for therapeutic transplantation using unsupervised/supervised learning.
- Provided strategic input and scientific advice.

`2009-Jun2012`
__Alm Laboratory for Microbiology, Prof. Eric J. Alm__, MIT

- [Supervised classification of microbiome samples](http://nbviewer.jupyter.org/github/elipapa/SLiMEbook/blob/master/SLiMEbook.ipynb) from pediatric IBD patients
- Feature extraction using ecological and phylogenetic information
- Quality control and downstream analysis of Human Microbiome Project sequencing pipeline

`2009`
__Founder, [Enumeral biomedical](http://www.enumeral.com/),__ Cambridge, MA

- MIT $100K Entrepeneurship competition semifinalist
- Contributed to the development of the microfluidic platform at the core of the company intellectual property

`2006-2009`
__Laboratory of Hidde L. Ploegh__, Whitehead Institute, MIT

- Unsupervised learning to map affinity and isotype of secreted antibodies in individual primary B cells.
- Describing immune response progression by statistical modelling of the B cell population
- Created a cluster pipeline to automate image analysis of fluorescence microscopy data

## Education

`2013`
__MBBS, Imperial College London__
Medicine & Surgery

`2012`
__Ph.D, [Harvard/MIT Health Science & Technology Institute](http://hst.mit.edu)__
Medical Engineering & Medical Physics

- Thesis: [Applied machine learning for exploring immunity and the microbiome](https://speakerdeck.com/u/elipapa/p/phd-defense-talk-1)

`2008`
__S.M., Massachusets Institute of Technology__
Mechanical Engineering

`2005`
__BASc (Honors), University of Toronto__
Engineering Science, Biomedical Option

## Publications

<!-- A list is also available [online](http://scholar.google.co.uk/citations?user=LTOTl0YAAAAJ) -->

### Journals

`2020`
Ochoa, D., Hercules, A., Carmona, M., Suveges, D., Gonzalez-Uriarte, A., Malangone, C., Miranda, A., Fumis, L., Carvalho-Silva, D., Spitzer, M., Baker, J., Ferrer, J., Raies, A., Razuvayevskaya, O., Faulconbridge, A., Petsalaki, E., Mutowo, P., Machlitt-Northen, S., Peat, G., McAuley, E., Ong, C.K., Mountjoy, E., Ghoussaini, M., Pierleoni, A., **Papa, E.**, Pignatelli, M., Koscielny, G., Karim, M., Schwartzentruber, J., Hulcoop, D.G., Dunham, I., & McDonagh, E.M. (2020). Open Targets Platform: supporting systematic drug-target identification and prioritisation. Nucleic acids research.

`2020`
Ghoussaini, M., Mountjoy, E., Carmona, M., Peat, G., Schmidt, E., Hercules, A., Fumis, L., Miranda, A., Carvalho-Silva, D., Buniello, A., Burdett, T., Hayhurst, J.D., Baker, J., Ferrer, J., Gonzalez-Uriarte, A., Jupp, S., Karim, M., Koscielny, G., Machlitt-Northen, S., Malangone, C., Pendlington, Z.M., Roncaglia, P., Suveges, D., Wright, D., Vrousgou, O., **Papa, E.**, Parkinson, H., MacArthur, J.A., Todd, J., Barrett, J., Schwartzentruber, J., Hulcoop, D.G., Ochoa, D., McDonagh, E.M., & Dunham, I. (2020). Open Targets Genetics: systematic identification of trait-associated genes using large-scale genetics and functional genomics. Nucleic acids research.

`2020`
Jackson, R.J., Jansson, E., Lagerberg, A., Ford, E., Poroshin, V., Scrivener, T., Axelsson, M., Johansson, M., Franco, L.A., & **Papa, E.** (2020). Ablations over transformer models for biomedical relationship extraction. F1000Research, 9, 710.

`2018`
Carvalho-Silva, D., Pierleoni, A., Pignatelli, M., Ong, C.K., Fumis, L., Karamanis, N., Carmona, M., Faulconbridge, A., Hercules, A., McAuley, E., Miranda, A., Peat, G., Spitzer, M., Barrett, J., Hulcoop, D.G., **Papa, E.**, Koscielny, G., & Dunham, I. (2019). Open Targets Platform: new developments and updates two years on. Nucleic Acids Research, 47, D1056 - D1065.

`2017`
Karamanis, N., Carvalho-Silva, D., Cham, J.A., Fumis, L., Hasan, S., Hulcoop, D.G., Koscielny, G., Maguire, M., Newell, W., Ong, C., **Papa, E.**, Pierleoni, A., Pignatelli, M., Pundir, S., Rowland, F., Vamathevan, J., Watkins, X., Barrett, J.C., & Dunham, I. (2017). Designing an intuitive web application for drug discovery scientists. bioRxiv.

`2017`
Koscielny G, An P, Carvalho-Silva D, Cham JA, Fumis L, Gasparyan R, Hasan S,
Karamanis N, Maguire M, **Papa E**, Pierleoni A, Pignatelli M, et al.
[Open Targets: a platform for therapeutic target identification and validation.](https://www.ncbi.nlm.nih.gov/pubmed/27899665)
*Nucleic Acids Res*. 2017 Jan 4;45(D1):D985-D994.
doi: 10.1093/nar/gkw1055. PMC5210543.

`2017`
Fischer M, Bittar M, **Papa E**, Kassam Z, Smith M.
[Can You Cause Inflammatory Bowel Disease with Fecal Transplantation? A 31-Patient Case-Series of Fecal Transplantation Using Stool from a Donor Who Later Developed Crohn's disease](http://www.tandfonline.com/doi/abs/10.1080/19490976.2017.1283469?journalCode=kgmi20)
 *Gut Microbes* 2017 Jan 19:0. doi: 10.1080/19490976.2017.1283469

`2012`
**Papa E**, Docktor M, Smillie C, Weber S, Preheim SP, Gevers D, Giannoukos G, Ciulla D, Tabbaa D, Ingram J, Schauer DB, Ward DV, Korzenik JR, Xavier RJ, Bousvaros A, Alm EJ.
[Non-invasive mapping of the gastrointestinal microbiota identifies children with inflammatory bowel disease.](http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0039242)
*PLoS ONE* 2012;7(6):e39242.

`2011`
White R, Miyata S, **Papa E**, Spooner E, Gounaris K, Selkirk M, Artavanis-Tsakonas K.
Characterisation of the Trichinella spiralis deubiquitinating enzyme, TsUCH37, an evolutionarily conserved proteasome interaction partner.
*PLoS Negl Trop Dis.* 2011 Oct;5(10):e1340.

`2011`
Artavanis-Tsakonas K, Kasperkovitz PV, **Papa E**, Cardenas ML, Khan NS, Van der Veen AG, Ploegh HL, Vyas JM.
The Tetraspanin CD82 is Specifically Recruited to Fungal and Bacterial Phagosomes Prior to Acidification.
*Infection and Immunity* 2011 79(3):1098-106\

`2009`
Adebola Ogunniyi A, Craig Story CM, **Papa E**, Guillen E, Love JC.
Screening Individual Hybridomas by Microengraving to Discover Monoclonal Antibodies.
*Nature Protocols* 2009 4(5):767-82

`2009`
Ronan JL, Story CM, **Papa E**, Love JC.
Optimization of the surfaces used to capture antibodies from single hybridomas reduces the time required for microengraving.
*Journal of Immunological Methods* 2009, 340(2):164-9\

`2008`
**Papa E**, Story CMâ—‡, Hu CC, Ronan JL, Herlihy K, Ploegh HL, Love JC.
[Profiling Antibody Responses by Multiparametric Analysis of Single B Cells.](http://www.pnas.org/content/105/46/17902.full)
*PNAS* 2008 105(46):17902-7

### Patents

`2009`
Composition of an Array of Microwells with an Integrated Microfluidic System, [US8569046B2](https://patents.google.com/patent/US8569046B2/en?inventor=eliseo+papa&oq=eliseo+papa)

## Fellowships

`2010-2011`
NSERC Postgraduate D Scholarship, Canada

`2008-2009`
Poitras pre-doctoral fellowship, MIT

`2007`
Martino Scholar, Harvard/MIT Health Science & Tech. Inst.

## Awards

`2008`
Martha Gray Prizes for Excellence in Research, Annual Forum, Harvard/MIT Health Science & Tech. Inst.

`2008`
Competition Semifinalist, MIT 100k Business Plan

## Volunteering

`2005`
Uganda Field operative, __[AISPO](http://www.aispo.org/), San Raffaele Foundation.__ Milan, Italy

`2004`
Conference delegate, __Engineers Without Borders__
