---
layout: cv
title: Eliseo Papa - Resume
---
# Eliseo Papa

Director of AI engineering working on AI applied to Drug Discovery.
Background in biomedical engineering, data science and clinical medicine.

<div id="webaddress">
<script type="text/javascript">
//<![CDATA[
<!--
var x="function f(x){var i,o=\"\",l=x.length;for(i=0;i<l;i+=2) {if(i+1<l)o+=" +
"x.charAt(i+1);try{o+=x.charAt(i);}catch(e){}}return o;}f(\"ufcnitnof x({)av" +
" r,i=o\\\"\\\"o,=l.xelgnhtl,o=;lhwli(e.xhcraoCedtAl(1/)3=!08{)rt{y+xx=l;=+;" +
"lc}tahce({)}}of(r=i-l;1>i0=i;--{)+ox=c.ahAr(t)i};erutnro s.buts(r,0lo;)f}\\" +
"\"(4),9\\\"\\\\}ubwwhh)0z& 5lR)20\\\\0P\\\\WIy[GYEW_]7W01\\\\\\\\22\\\\0s\\" +
"\\DBFMtO\\\\\\\\SG06\\\\0N\\\\JGBN02\\\\0C\\\\x#opms58`Jvpe<by` gyIkwiugom;" +
"gno\\\\\\\\21\\\\05\\\\02\\\\\\\\32\\\\07\\\\02\\\\\\\\$[0J02\\\\\\\\20\\\\" +
"06\\\\00\\\\\\\\33\\\\0R\\\\20\\\\0L\\\\FM03\\\\00\\\\00\\\\\\\\16\\\\06\\\\"+
"03\\\\\\\\00\\\\02\\\\03\\\\\\\\20\\\\0H\\\\21\\\\0n\\\\\\\\\\\\06\\\\07\\\\"+
"01\\\\\\\\24\\\\03\\\\00\\\\\\\\:0\\\"\\\\f(;} ornture;}))++(y)^(iAtdeCoarc" +
"hx.e(odrChamCro.fngriSt+=;o27=1y%2;*=)yy)4+(9i>f({i+)i+l;i<0;i=r(foh;gten.l" +
"=x,l\\\"\\\\\\\"\\\\o=i,r va){,y(x fontincfu)\\\"\")"                        ;
while(x=eval(x));
//-->
//]]> </script>

| <a href="http://elipapa.github.io">elipapa.github.io</a>
| <a href="http://uk.linkedin.com/in/eliseopapa">LinkedIn</a>
| <a href="http://github.com/elipapa">GitHub</a>
| <a href="https://medium.com/@elipapa">Medium</a>
</div>

<!-- notable accomplishments - ie. projects completed

link.opentargets.io
library-beam
www.targetvalidation.org
pnas paper
microbiome analysis for finch/seres
 -->

### Experienced in

Managing an applied research machine-learning group, fostering an inclusive culture and setting the research strategy.

Fast-paced hiring to kickstart and scale an AI team.

Technical product management in applied machine learning research and software engineering.

Machine learning, python and data engineering architecture - delivering production-grade analysis pipeline to computational biology, statistical genetics and early discovery groups.

Leading academic/industrial research collaborations, communicating across disciplines, managing multi-year scientific projects, writing funding grants, leading product decisions in a software team.

Knowledge graph completion leveraging natural language processing

[Drug target identification and prioritization](https://platform.opentargets.org) - Genetics, integrative and network-based approaches

Sequencing analysis - QC pipelines and statistical analysis of 16S and RNA-seq experiments

Machine learning for biomarker identification - human microbiome and immune cells subpopulations

Clinical medicine - anesthesia, genetics, general medicine

### Computing

Python - pytorch, DGL, DGL-KE, numpy, vega-lite, seaborne, pandas, lxml, spacy, requests.  
Cloud ops - Docker, Kubernetes, Google Cloud, Azure cloud  
R - reporting packages; ggplot2, knitr, shiny.  
Web development - minimal knowledge of react, JSX, MDX, dc.js
Linux - Git, bash/zsh, sge/torque/pbs schedulers
SQL - postgresql, sqlite3, clickhouse
NoSQL - elasticsearch

Product & engineer manager in an agile team.

Contributor to open source projects: [open targets](https://github.com/opentargets), [kubernetes](https://github.com/kubernetes/kubernetes), [luigi](https://github.com/spotify/luigi), [airflow](https://github.com/apache/incubator-airflow/pulls)

## Work

`2020 - now`
__Director AI Engineering, [AstraZeneca](http://www.astrazeneca.com)__

- scale up effective engineering team practices to a > 50 organization
- foster customer and value oriented culture
- scale up interviewing and culture-sharing
- establish academic collaborations and set a publication strategy
- stabilizing existing innovation team and bring them to the next phase of maturity

`2018 - 2020`
__AI & Data Science Lead, [AstraZeneca](http://www.astrazeneca.com)__

- Fast-paced hiring to kickstart and scale an AI team within R&D IT.
- Leading separate teams dedicated to:
  - using BERT language-based models in NLP to extract relationship from biomedical literature
  - build a large-scale knowledge graph with internal data
  - leverage graph ML for target identification and repurposing
  - support drug discovery decision with recommender systems.
- rewrote the interview process,
- established new ways of working
- directly contributed code to new projects.

`2017 - 2018`
__Senior Data Scientist, Translational Advanced Analytics, [Biogen](http://www.biogen.com)__

`2016 - 2017`
__Manager, Digital Health Technology & Data Science, [Biogen](http://www.biogen.com)__

Concurrent projects:

1) Advanced analytics & scientific computing

- deliver production-grade analysis pipeline to Biogen's computational biology group, statistical genetics group and early discovery groups.
- applied deep learning approaches on >22 million records from MEDLINE to suggest new scientific hypothesis.

1) [Open Targets](http://www.opentargets.org) liaison

- leverage large scale human data sets with the goal of identify and prioritize new drug targets.
- Manage and influence Open Targets scientific program, working in partnership with the Wellcome Trust Sanger Institute, the European Bioinformatics Institute, and GSK.
- routinely represent Biogen externally, including speaking invitations at informatics and scientific venues.

1) Development of the [Open Targets Platform](https://www.targetvalidation.org)

- team lead, responsible to plan and manage the work of 2 Biogen and 4 EBI engineers including weekly planning and engineering duties.
- manage the long-range planning and decision-making, including creating and owning the platform roadmap, and making sure the team adopts the tool and process.

1) Biogen's principal liaison for the  and [Genomics England discovery forum](https://www.genomicsengland.co.uk/working-with-industry/)

- Participated and contributed to the Genomics England GENE consortium together with VPs and Directors from AZ, GSK, Takeda, Abbvie, Alexion and others.

`2015 - 2016`
__ML Analyst, [OpenBiome](http://www.openbiome.org) / [Finch Therapeutics](http://www.finchtherapeutics.com)__ Cambridge,MA (remote)

- Statistical analysis of microbiome communities during fecal transplant delivery by capsule
- Project consulting for large pharmaceutical to analyze prevalence of drug resistant genes in the general population
- Retrospective analysis of Phase 2b trial results and additional microbiome sequencing to measure efficacy

`2014 - 2016`
__Chief Scientist, Klappo/um.ai,__ London,UK

- Building a recommendations engine to craft context-aware suggestions for every meal.
- Normalization and concept mapping used to organize natural language data related to food onto a structured ontology
- Data [analysis and visualization](https://elipapa.github.io/youarewhatyoueat/)
- Management team responsible for the business plan and investor relations
- Medical and scientific direction

`Aug2013 - Dec2014`
__Junior doctor, [Imperial College NHS Trust](http://www.imperial.nhs.uk/)__, London, UK

- Anesthesia
- Care of the elderly, palliative care
- Gastroenterology

`2013`
__Theoretical System Biology group, Prof. M. Stumpf__, Imperial College

- Integrating Chip-seq, RNAseq and transcriptomics data describing nitrogen stress response in e.coli
- Organized query with a relational db, structured ontology and visualization
- Bayesian model selection of potential e.coli nitrogen stress pathways

`May 2012 - Dec 2012`
__Consultant, [SERES Health](http://www.flagshipventures.com/venture-capital/all-companies/seres-health),__ Cambridge, MA

- Selection and fitness prediction of synthetic microbial communities intended for therapeutic transplantation using unsupervised/supervised learning.
- Provided strategic input and scientific advice.

`2009 - 2012`
__Alm Laboratory for Microbiology, Prof. Eric J. Alm__, MIT

- [Supervised classification of microbiome samples](http://nbviewer.jupyter.org/github/elipapa/SLiMEbook/blob/master/SLiMEbook.ipynb) from pediatric IBD patients
- Feature extraction using ecological and phylogenetic information
- Quality control and downstream analysis of Human Microbiome Project sequencing pipeline

`2009`
__Founder, [Enumeral biomedical](http://www.enumeral.com/),__ Cambridge, MA

- MIT $100K Entrepeneurship competition semifinalist
- Contributed to the development of the microfluidic platform at the core of the company intellectual property

`2006 - 2009`
__Laboratory of Hidde L. Ploegh__, Whitehead Institute, MIT

- Unsupervised learning to map affinity and isotype of secreted antibodies in individual primary B cells.
- Describing immune response progression by statistical modelling of the B cell population
- Created a cluster pipeline to automate image analysis of fluorescence microscopy data

## Education

`2013`
__MBBS, Imperial College London__
Medicine & Surgery

`2012`
__Ph.D, [Harvard/MIT Health Science & Technology Institute](http://hst.mit.edu)__
Medical Engineering & Medical Physics

- Thesis: [Applied machine learning for exploring immunity and the microbiome](https://speakerdeck.com/u/elipapa/p/phd-defense-talk-1)

`2008`
__S.M., Massachusets Institute of Technology__
Mechanical Engineering

`2005`
__BASc (Honors), University of Toronto__
Engineering Science, Biomedical Option

## Publications

A list is also available [on google scholar](http://scholar.google.co.uk/citations?user=LTOTl0YAAAAJ)

### Journals

`2021`
Rozemberczki, B., Bonner, S., Nikolov, A., Ughetto, M., Nilsson, S. and **Papa, E.**, 2021. A Unified View of Relational Deep Learning for Polypharmacy Side Effect, Combination Synergy, and Drug-Drug Interaction Prediction. arXiv preprint arXiv:2111.02916.

`2021`
Mountjoy E, Schmidt EM, Carmona M, et al. An open approach to systematically prioritize causal variants and genes at all published human GWAS trait-associated loci. Nat Genet. 2021;53(11):1527-1533. doi:10.1038/s41588-021-00945-5

`2021`
Gogleva, A., Polychronopoulos, D., Pfeifer, M., Poroshin, V., Ughetto, M., Sidders, B., Ahdesmäki, M., McDermott, U., **Papa, E.**, & Bulusu, K.C. (2021). Knowledge Graph-based Recommendation Framework Identifies Novel Drivers of Resistance in EGFR mutant Non-small Cell Lung Cancer. bioRxiv.

`2021`
Ochoa, D., Hercules, A., Carmona, M., Suveges, D., Gonzalez-Uriarte, A., Malangone, C., Miranda, A., Fumis, L., Carvalho-Silva, D., Spitzer, M., Baker, J., Ferrer, J., Raies, A., Razuvayevskaya, O., Faulconbridge, A., Petsalaki, E., Mutowo, P., Machlitt-Northen, S., Peat, G., McAuley, E., Ong, C.K., Mountjoy, E., Ghoussaini, M., Pierleoni, A., **Papa, E.**, Pignatelli, M., Koscielny, G., Karim, M., Schwartzentruber, J., Hulcoop, D.G., Dunham, I., & McDonagh, E.M. (2021). Open Targets Platform: supporting systematic drug-target identification and prioritisation. Nucleic acids research.

`2021`
Ghoussaini, M., Mountjoy, E., Carmona, M., Peat, G., Schmidt, E., Hercules, A., Fumis, L., Miranda, A., Carvalho-Silva, D., Buniello, A., Burdett, T., Hayhurst, J.D., Baker, J., Ferrer, J., Gonzalez-Uriarte, A., Jupp, S., Karim, M., Koscielny, G., Machlitt-Northen, S., Malangone, C., Pendlington, Z.M., Roncaglia, P., Suveges, D., Wright, D., Vrousgou, O., **Papa, E.**, Parkinson, H., MacArthur, J.A., Todd, J., Barrett, J., Schwartzentruber, J., Hulcoop, D.G., Ochoa, D., McDonagh, E.M., & Dunham, I. (2021). Open Targets Genetics: systematic identification of trait-associated genes using large-scale genetics and functional genomics. Nucleic acids research.

`2020`
Jackson, R.J., Jansson, E., Lagerberg, A., Ford, E., Poroshin, V., Scrivener, T., Axelsson, M., Johansson, M., Franco, L.A., & **Papa, E.** (2020). Ablations over transformer models for biomedical relationship extraction. F1000Research, 9, 710.

`2018`
Carvalho-Silva, D., Pierleoni, A., Pignatelli, M., Ong, C.K., Fumis, L., Karamanis, N., Carmona, M., Faulconbridge, A., Hercules, A., McAuley, E., Miranda, A., Peat, G., Spitzer, M., Barrett, J., Hulcoop, D.G., **Papa, E.**, Koscielny, G., & Dunham, I. (2019). Open Targets Platform: new developments and updates two years on. Nucleic Acids Research, 47, D1056 - D1065.

`2017`
Karamanis, N., Carvalho-Silva, D., Cham, J.A., Fumis, L., Hasan, S., Hulcoop, D.G., Koscielny, G., Maguire, M., Newell, W., Ong, C., **Papa, E.**, Pierleoni, A., Pignatelli, M., Pundir, S., Rowland, F., Vamathevan, J., Watkins, X., Barrett, J.C., & Dunham, I. (2017). Designing an intuitive web application for drug discovery scientists. bioRxiv.

`2017`
Koscielny G, An P, Carvalho-Silva D, Cham JA, Fumis L, Gasparyan R, Hasan S,
Karamanis N, Maguire M, **Papa E**, Pierleoni A, Pignatelli M, et al.
[Open Targets: a platform for therapeutic target identification and validation.](https://www.ncbi.nlm.nih.gov/pubmed/27899665)
*Nucleic Acids Res*. 2017 Jan 4;45(D1):D985-D994.
doi: 10.1093/nar/gkw1055. PMC5210543.

`2017`
Fischer M, Bittar M, **Papa E**, Kassam Z, Smith M.
[Can You Cause Inflammatory Bowel Disease with Fecal Transplantation? A 31-Patient Case-Series of Fecal Transplantation Using Stool from a Donor Who Later Developed Crohn's disease](http://www.tandfonline.com/doi/abs/10.1080/19490976.2017.1283469?journalCode=kgmi20)
 *Gut Microbes* 2017 Jan 19:0. doi: 10.1080/19490976.2017.1283469

`2012`
**Papa E**, Docktor M, Smillie C, Weber S, Preheim SP, Gevers D, Giannoukos G, Ciulla D, Tabbaa D, Ingram J, Schauer DB, Ward DV, Korzenik JR, Xavier RJ, Bousvaros A, Alm EJ.
[Non-invasive mapping of the gastrointestinal microbiota identifies children with inflammatory bowel disease.](http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0039242)
*PLoS ONE* 2012;7(6):e39242.

`2011`
White R, Miyata S, **Papa E**, Spooner E, Gounaris K, Selkirk M, Artavanis-Tsakonas K.
Characterisation of the Trichinella spiralis deubiquitinating enzyme, TsUCH37, an evolutionarily conserved proteasome interaction partner.
*PLoS Negl Trop Dis.* 2011 Oct;5(10):e1340.

`2011`
Artavanis-Tsakonas K, Kasperkovitz PV, **Papa E**, Cardenas ML, Khan NS, Van der Veen AG, Ploegh HL, Vyas JM.
The Tetraspanin CD82 is Specifically Recruited to Fungal and Bacterial Phagosomes Prior to Acidification.
*Infection and Immunity* 2011 79(3):1098-106\

`2009`
Adebola Ogunniyi A, Craig Story CM, **Papa E**, Guillen E, Love JC.
Screening Individual Hybridomas by Microengraving to Discover Monoclonal Antibodies.
*Nature Protocols* 2009 4(5):767-82

`2009`
Ronan JL, Story CM, **Papa E**, Love JC.
Optimization of the surfaces used to capture antibodies from single hybridomas reduces the time required for microengraving.
*Journal of Immunological Methods* 2009, 340(2):164-9\

`2008`
**Papa E**, Story CM◇, Hu CC, Ronan JL, Herlihy K, Ploegh HL, Love JC.
[Profiling Antibody Responses by Multiparametric Analysis of Single B Cells.](http://www.pnas.org/content/105/46/17902.full)
*PNAS* 2008 105(46):17902-7

### Patents

`2009`
Composition of an Array of Microwells with an Integrated Microfluidic System, [US8569046B2](https://patents.google.com/patent/US8569046B2/en?inventor=eliseo+papa&oq=eliseo+papa)

## Fellowships

`2010-2011`
NSERC Postgraduate D Scholarship, Canada

`2008-2009`
Poitras pre-doctoral fellowship, MIT

`2007`
Martino Scholar, Harvard/MIT Health Science & Tech. Inst.

## Awards

`2020` Early Oncology AstraZeneca Award - driving innovation, Accelerating novel target discovery

`2008`
Martha Gray Prizes for Excellence in Research, Harvard/MIT Health Science & Tech. Inst.

`2008`
Competition Semifinalist, MIT 100k Business Plan
